py/pybtls/bridge/bridge.py

```diff
@@ -97,10 +97,17 @@ def add_load_effect(
                     "The number of influence lines should be equal to the number of lanes."
                 )
 
+            if not all([inf_line._data_assigned for inf_line in inf_line_surf]):
```

Comment:
It would be better if these code changes were not included in a PR that redesigns the build system. Can you put these into a separate PR?

Reply:
I will make changes based on your comments, then close this PR and create three new PRs.